als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/plugins/aml/webapi/raml/PayloadMediaTypeSeeker.scala

```diff
@@ -31,6 +31,7 @@ trait PayloadMediaTypeSeeker {
             .option()
             .isEmpty && inMediaType(request.yPartBranch)
         case Some(_: Operation) => inMediaType(request.yPartBranch)
+        case Some(_: Request)   => inMediaType(request.yPartBranch)
```

Comment:
which test is this affecting?

Reply:
the brachStack head for a second mime si a Request type, that's why the case is added. Prior to this change, the plugin responsible for the return of the suggestion (`RamlPayloadMediaTypeCompletionPlugin`) returned nothig.